code/modules/synd_specialist/specialist.dm

```diff
@@ -0,0 +1,267 @@
+//The syndicate elite
+//I have no business making something this advanced
+
+
+//Trying to keep it all in one file
+/*----------------
+     CLOTHING
+----------------*/
+
+/obj/item/clothing/suit/agent_coat//the coat
+	name = "Trenchcoat"
+	desc = "A fashionable black leather trenchcoat. Supposedly inconspicuous."
+	allowed = /obj/item/weapon/gun //They can only carry guns in their coat
+	icon_state = "hos"
+	item_state = "greatcoat"
+	slowdown = 0
+	unacidable = 1
+	armor = list(melee = 85, bullet = 70, laser = 70, energy = 0, bomb = 30, bio = 0, rad = 0)
+	strip_delay = INFINITY//No, you can't have it
+
+/obj/item/clothing/under/syndicate/agent_suit
+	name = "Black Ops suit"
+	desc = "Nanotrasen sure pissed off somebody important."
+	strip_delay = INFINITY
+
+
+/*----------------
+    ABILITIES
+----------------*/
+/obj/effect/proc_holder/spell/proc/agent_check(var/mob/living/carbon/human/H)//shamelessly stolen from sling
+	if(!H || !istype(H)) return
+	if(H.dna.species.id == "agent") return 1
+	if(!H.dna.species.id== "agent") usr << "You aren't augmented enough to do this."
```

Comment:
you don't even need an else statement; this won't be reached if this check would fail